src/main/java/cc/hyperium/gui/ChangeBackgroundGui.java

```diff
@@ -28,7 +44,7 @@
 public class ChangeBackgroundGui extends GuiScreen {
 
     private final GuiScreen prevGui;
-
+    private static final boolean isFS = Minecraft.getMinecraft().isFullScreen();
```

Comment:
This shouldn't be static or final. The user can change whether they're on fullscreen mode while the GUI is open. Instead, call Minecraft.getMinecraft().isFullScreen() in each of the methods where this is used, or assign it to a variable within the methods where it is used multiple times.

Reply:
👍